source/FilesServer/pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.poetry]
+name = "files-server"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.11"
+fastapi = "^0.110.0"
+beanie = "^1.25.0"
+uvicorn = "^0.29.0"
+pydantic = "^2.6.4"
+pydantic-settings = "^2.2.1"
+ruff = "^0.3.4"
```

Comment:
ruff should be a development dependency.
remove it with "poetry remove ruff" and reinstall  it with
"poetry add ruff -G dev